docs/hooks/order-review.md

```diff
@@ -19,6 +19,186 @@ Field | Optionality | Prefetch Token | Type | Description
 `encounterId` | OPTIONAL | Yes | *string* | The FHIR `Encounter.id` of the current encounter in context
 `orders` | REQUIRED | No | *object* | DSTU2 - FHIR Bundle of MedicationOrder, DiagnosticOrder, DeviceUseRequest, ReferralRequest, ProcedureRequest, NutritionOrder, VisionPrescription with _draft_ status <br/> STU3 - FHIR Bundle of MedicationRequest, ReferralRequest, ProcedureRequest, NutritionOrder, VisionPrescription with _draft_ status
 
+### Example (STU3)
+
+```json
+{
+  "context":{
+    "patientId":"1288992",
+    "encounterId":"89284",
+    "orders":{
+      "resourceType":"Bundle",
+      "entry":[
+        {
+          "resource":{
+            "resourceType":"NutritionOrder",
+            "id":"pureeddiet-simple",
+            "identifier":[
+              {
+                "system":"http://goodhealthhospital.org/nutrition-requests",
+                "value":"123"
+              }
+            ],
+            "status":"draft",
+            "patient":{
+              "reference":"Patient/1288992",
```

Comment:
This trailing comma needs to be removed in order for the JSON to be compliant.

Reply:
Thanks, kdog! Fixed.

I only saw the one comma. 

What do you think?

Reply:
@isaacvetter - The second trailing comma is in order-review.md on line 44:

```"reference":"Patient/1288992",```

The fix you made to medication-prescribe.md is good!